readmestaging20180812

```diff
@@ -0,0 +1 @@
+Creating branch staging20180812
```

Comment:
cleanup

Reply:
removed -- it got in here because I merged WFP's staging into my branch